src/views/abteilung/material/abteilungMaterials.tsx

```diff
@@ -124,6 +127,18 @@ export const AbteilungMaterialView = (props: AbteilungMaterialViewProps) => {
                 <AddStandortButton abteilungId={abteilung.id} />
             </Can>
         </Col>
+        <Can I={'delete'} this={{ __caslSubjectType__: 'Material', abteilungId: abteilung.id } as AbteilungEntityCasl}>
```

Comment:
Ich bin nicht wirklich ein Fan von den Buttons unter der Navigation. Gibt es da einen besseren Weg? 
Habt ihr dazu eine Meinung @jan-preisig @JeremiasBachmann @pemko123 ?

Reply:
UM alles Material zu löschen denke ich sollte dies beim Excel Import sein oder nicht?
Gehört irgendwie zusammen. Wann sonst sollte man alles MAterial löschen?

Reply:
Wie wäre es mit einem Dropdown in der Navigation namens Aktionen von wo aus man verschiedene Aktionen ausführen kann? Entweder direkt aus dem Dropdwon das popup öffnen oder eine Subpage mit funktionen?

Oder Option 2 wäre man nimmt alles zu den Einstellungen und hat dort einerseits funkionen um diverse dinge zu Bearbeiten und Erstellen sowie die Einstellungen.

Findet ihr davon etwas gut? @JeremiasBachmann @Leo1212 @pemko123

Reply:
Aktionen in der Navigation könnte man machen, solange es nicht zu viele werden. Unter Settings fände ich die Buttons irgendwie nicht sehr intuitiv, da Ich Buttons welche mit dem Material zu tun haben unter dem Menupunkt Material erwarten würde und nicht bei Settings. Ich finde die Buttons aber eigentlich nicht schlecht platziert, vor allem da sie ohnehin nur für das MatTeam/Admins sichtbar sind. Eine weiter Möglichkeit wäre das man für das MatTeam einen eigenen Menupunkt erstellt, in welchem sie das Mat verwalten können, und MatTeam spezifische Einstellung wie  z.B. das aktivieren von Email Benachrichtigungen etc. haben.

Reply:
> UM alles Material zu löschen denke ich sollte dies beim Excel Import sein oder nicht? Gehört irgendwie zusammen. Wann sonst sollte man alles MAterial löschen?

Beim Excel Import wurde dies so umgesetzt, dass man wählen kann ob das neue Material, das alte Material ersetzten soll oder ob es zum alten Material hinzugefügt werden soll.